src/commons/api/roas.rs

```diff
@@ -96,6 +95,23 @@ impl<'de> Deserialize<'de> for RoaAggregateKey {
     }
 }
 
+/// Ordering is based on the ordering implemented by RoaDefinition
```

Comment:
Is it? It seems to be based on the ordering implemented by AsNumber rather than ROADefinition?